tests/integration/auth.py

```diff
@@ -235,6 +236,11 @@ async def _factory(
                         cluster_name=cluster,
                         org_name=org_name,
                     )
+                    await admin_client.update_org_cluster_balance(
```

Comment:
👍🏼 
Could you use new new `update_org_balance` method instead?
Curious why it wasn't failing before

Reply:
Not at the moment, we need to patch jobs service to verify org balance first.

At the moment, we verify only ClusterUser's | OrgClusterUser's and OrgCluster's balance before submitting the job https://github.com/neuro-inc/platform-api/blob/master/platform_api/orchestrator/jobs_service.py#L258-L276

and on a runtime
https://github.com/neuro-inc/platform-api/blob/master/platform_api/orchestrator/job_policy_enforcer.py#L197-L222

I would suggest patching it in a dedicated PR

re before -- we previously created orgs with None credits balance by default, but now -- with '0': https://github.com/neuro-inc/platform-admin/pull/1025/files#diff-716490342b5edb6e7706df6e5a656487859d64760cfe80bf6095c39acfa597cdR164